packages/gestalt-datepicker/.npmignore

```diff
@@ -0,0 +1,2 @@
+**/__*__/**
```

Comment:
Just wondering what this path is here to match?

Reply:
This was on /gestalt
I'm don't know what this is used for. Maybe can be removed

Reply:
I read it as files that are in the form of `__FOO__`. Looks like `__snapshots__` fit that description.

Reply:
it feels to me like it's doing nothing, but not sure

Reply:
it was to remain consistent but we no longer have these directories

![image](https://user-images.githubusercontent.com/127199/84550647-02450f00-acc0-11ea-9cb1-15e562ed1b70.png)